src/main/java/edu/ksu/canvas/net/SimpleRestClient.java

```diff
@@ -258,9 +264,9 @@ private void checkHeaders(HttpResponse httpResponse, HttpRequestBase request) {
      */
     private String extractErrorMessageFromResponse(HttpResponse response) {
         String contentType = response.getEntity().getContentType().getValue();
+        String responseBody = null;
```

Comment:
Consider initializing this with an empty string rather than null.
This could lead to an unexpected NullPointerException further on down the line.

Reply:
Just so long as the method still returns null if no error message can be extracted :)